src/main/java/printcount/PrintCount.java

```diff
@@ -0,0 +1,21 @@
+package printcount;
+
+public class PrintCount {
+
+
+    public void CountResult(int ball, int strike){
+
+        if (ball == 0 && strike ==0){
+            System.out.println("낫싱");
+        } else if(ball == 0 && strike != 0 ) {
+            System.out.printf("%d스트라이크", strike);
+            System.out.println();
+        } else if(ball != 0 && strike == 0 ) {
+            System.out.printf("%d볼", ball);
+            System.out.println();
+        } else {
+            System.out.printf("%d볼 %d스크라이크",ball,strike);
```

Comment:
랜덤값 이후의 테스트 코드에서 출력 테스트가 실패를 했는데요.
아래와 같이 스크라이크 에서 오타가 발생해서, '스트라이크' 관련 출력 테스트가 실패했습니다.

```suggestion
            System.out.printf("%d볼 %d스트라이크",ball,strike);
```

Reply:
헉.... 이런 실수가 있었다니... 정말 제 코드 꼼꼼히 봐주셨군요... 감사합니다!! ㅜㅜ